#starting IDs corresponding with the cf_ID

int32[] starting_IDs
